mava/systems/tf/mappo/builder.py

```diff
@@ -177,10 +245,19 @@ def make_dataset_iterator(
         """
 
         # Create tensorflow dataset to interface with reverb
-        dataset = datasets.make_reverb_dataset(
+        # dataset = datasets.make_reverb_dataset(
+        #     server_address=replay_client.server_address,
+        #     batch_size=self._config.batch_size,
+        # )
+
+        dataset = reverb.TrajectoryDataset.from_table_signature(
             server_address=replay_client.server_address,
-            batch_size=self._config.batch_size,
-        )
+            table=table_name,
+            max_in_flight_samples_per_worker=1,
```

Comment:
Might be a good idea to set this to 2 * batch size following acme - https://github.com/deepmind/acme/blob/93a6a15b8c7b1d6a9c8099b9f2e9c693ab11a41e/acme/agents/jax/ppo/builder.py#L97 .

Reply:
Good point. Do you know what `max_in_flight_samples_per_worker` represents?

Reply:
Found the readme here - https://github.com/deepmind/reverb/blob/b57722c77cb5f36f82688aae8502c23a7a8de6bd/reverb/dataset.py#L76 .

Reply:
Fixed.